src/app/features/management/user-management-view/create-user-view/create-user-view.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="view" [appShowLoadingOverlay]="creatingUser.loadingChange() | async">
+  <h1 i18n>Create User</h1>
+  <form (submit)="createUser()" class="example-form">
+    <mat-form-field class="app-edit-input">
+      <mat-label i18n>Username</mat-label>
+      <input matInput [formControl]="form.controls.username">
+      <mat-error *ngIf="form.controls.username.errors && form.controls.username.errors['required']" i18n>Username is
+        required.
+      </mat-error>
+    </mat-form-field>
+    <mat-form-field class="app-edit-input">
+      <mat-label i18n>First Name</mat-label>
+      <input matInput [formControl]="form.controls.firstName">
+      <mat-error *ngIf="form.controls.firstName.errors && form.controls.firstName.errors['required']" i18n>First name is
+        required.
+      </mat-error>
+    </mat-form-field>
+    <mat-form-field class="app-edit-input">
+      <mat-label i18n>Last Name</mat-label>
+      <input matInput [formControl]="form.controls.lastName">
+      <mat-error *ngIf="form.controls.lastName.errors && form.controls.lastName.errors['required']" i18n>Last name is
+        required.
+      </mat-error>
+    </mat-form-field>
+    <mat-form-field class="app-edit-input">
+      <mat-label i18n>Initial Password</mat-label>
+      <input matInput type="password" [formControl]="form.controls.pass">
+      <mat-error *ngIf="form.controls.pass.errors && form.controls.pass.errors['required']" i18n>Password is required.
+      </mat-error>
+    </mat-form-field>
+    <mat-checkbox [formControl]="form.controls.isAdmin">Is Admin</mat-checkbox>
+    <div class="button-row">
+      <button mat-button type="button" (click)="cancel()" i18n>
+        Cancel
+      </button>
+      <button mat-button color="primary" type="submit"
+              [disabled]="!form.valid || (creatingUser.loadingChange() | async)" i18n>
```

Comment:
- [ ] see comments for edit-view
- [ ] german translation would not be _Nutzer Erstellen_ but _Nutzer erstellen_. Nonetheless, rename to _Create_/_Erstellen_.